check_clang_tidy.sh

```diff
@@ -4,16 +4,16 @@
 
 _travis_run_clang_tidy_fix() {
     local SOURCE_PKGS COMPILED_PKGS counter pkg file
-    SOURCE_PKGS=($(catkin_topological_order $CI_SOURCE_PATH --only-names 2> /dev/null))
+    SOURCE_PKGS=()
 
     # filter repository packages for those which have a compile_commands.json file in their build folder
     declare -A PKGS  # associative array
-    for pkg in ${SOURCE_PKGS[@]} ; do
+    for pkg in $(colcon info | grep 'name: ' | sed -e "s/.*name: //g" 2> /dev/null) ; do
```

Comment:
I think `SOURCE_PKGS` should be initialized from this command. Otherwise `SOURCE_PKGS` is not used anymore and could be deleted.

Reply:
Good find